### PR TITLE
Feature/1815 display errors in conditional expression

### PR DESCRIPTION
**Pull request by stevenburnell-moj**
Show/control error message for unsupported BranchQuestion

Screenshot 2021-08-17 at 01 18 03

### CODE REVIEW

File: app/views/partials/_properties.html.erb [hunk 1]
```diff
@@ -20,6 +20,11 @@
         question: "<%= t('aria.question') %>",
         section_heading: "<%= t('aria.section_heading') %>"
       },
+      errors: {
+        branches: {
+          unsupported_question: "<%= t('activemodel.errors.messages.unsupported') %>"
```
Comment (brenetic) [inline app/views/partials/_properties.html.erb:25]: if it's being grabbed from the locales can we therefore remove it from the data attribute on the div?

**Review by brenetic — DISMISSED** [review]
🚀

**Comment by brenetic** [comment]
@stevenburnell-moj great stuff! just the one question

File: app/views/partials/_properties.html.erb (hunk 1 above)
Comment (stevenburnell-moj) [reply app/views/partials/_properties.html.erb:25]: Yes.
Sing it... dum, dum, dum... Another attribute bites the dust!

Comment (stevenburnell-moj) [reply app/views/partials/_properties.html.erb:25]: PR should now have commit with removal of attr.

**Review by brenetic — APPROVED** [review]
🚀

**Pull request merged**